lib/screens/about.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
+
+class AboutPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('About Umami'),
+      ),
+      body: Text(
+        "Hello About Info!"
+      ),
+    );
+  }
+}
```

Comment:
it appears to be wanting a newline here.

Reply:
Added new line and some other code formatting fixes for Dart files. Thank you!